src/FLOWVPM_viscous.jl

```diff
@@ -398,33 +398,33 @@ function rbf_conjugategradient(pfield, cs::CoreSpreading)
                     "\t"^(cs.v_lvl+1)*"Iteration $(it) / $(cs.itmax) max\n"*
                     "\t"^(cs.v_lvl+2)*"Error: $(sqrt.(cs.rrs ./ cs.rr0s))\n"*
                     "\t"^(cs.v_lvl+2)*"Flags: $(cs.flags)\n"*
-                    "\t"^(cs.v_lvl+2)*"Sol Particle 1: $(round.(get_particle(pfield, 1).M[1:3], digits=8))"
-                    )
+                    "\t"^(cs.v_lvl+2)*"Sol Particle 1: $(round.(get_particle(pfield, 1)[28:30], digits=8))"
+                   )
         end
 
     end
 
     # Save final solution
     for P in iterator(pfield)
         for i in 1:3
-            P.Gamma[i] = P.M[i]
+            get_Gamma(P)[i] = get_M(P)[i]
         end
     end
 
     if cs.debug
         # Evaluate current vorticity
         cs.zeta(pfield)
         println("\t"^(cs.v_lvl+1)*"***** Probe Particle 1 ******\n"*
-                "\t"^(cs.v_lvl+2)*"Final Gamma:\t$(round.(get_particle(pfield, 1).Gamma, digits=8))\n"*
+                "\t"^(cs.v_lvl+2)*"Final Gamma:\t$(round.(get_particle(pfield, 1)[4:6], digits=8))\n"*
```

Comment:
Use getter and setter functions here instead of indexing

Reply:
Resolved in [f1993eb](https://github.com/byuflowlab/FLOWVPM.jl/pull/13/commits/f1993ebff28329e265be6bd8c09b7cbfeab65610)